core/node/da_clients/Cargo.toml

```diff
@@ -50,7 +50,7 @@ sha2.workspace = true
 prost.workspace = true
 bech32.workspace = true
 ripemd.workspace = true
-tonic = { workspace = true, features = ["tls", "default"] }
+tonic = { workspace = true, features = ["tls", "tls-roots", "prost", "codegen"] }
```

Comment:
> tls-roots: Deprecated. An alias to tls-native-roots feature.

Reply:
`tls-native-roots` is only available in newer versions